### DISCUSSION TITLE
Dropping support for Python 2.7

### DISCUSSION THREAD

**Discussion by tschutter** [netromdk/vermin#87] [Q&A] [answered]
The README states that "Vermin 1.5 will end support for Python 2.7", but it is not clear if this means that support for **running** under 2.7 is being dropped or if support for **detecting** 2.7 code is being dropped.

So which is it?  I have no problem requiring 3.x to run vermin.  It would be nice if detection of 2.x was still supported.

**Comment by netromdk** [comment, marked as answer]
Thank you for the feedback. I realize the ambiguity now. The intent is to require Python 3.x to execute Vermin but the detection of Python 2.x functionality will not be removed. I'll update the phrasing of the README to be more clear.
